PowerEditor/src/Parameters.h

```diff
@@ -1648,6 +1648,14 @@ class NppParameters final
 	NppParameters();

```

Comment:
`NppParameters() = default;` ?

Reply:
Both of them can't be default as these already have user defined ctor and dtor.